common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -624,7 +624,7 @@ def definition_from_xml(cls, xml_object, system):
         ]
         definition = {
             attr_name: json.loads(attr_value)
-            for attr_name, attr_value in xml_object.attrib
+            for attr_name, attr_value in xml_object.attrib.iteritems()
```

Comment:
Do we need `iteritems()` optimization here? Can we get away with `items()` so we have one less thing to worry about with Python 3?